audio/engine.asm

```diff
@@ -1197,12 +1216,15 @@ ParseMusic:
 	set NOTE_REST, [hl] ; Rest
 	ret
 
+; $ff was read from music data
+; aka sound_ret_cmd
 .sound_ret
-; $ff is reached in music data
 	ld hl, CHANNEL_FLAGS1
 	add hl, bc
-	bit SOUND_SUBROUTINE, [hl] ; in a subroutine?
-	jr nz, .readcommand ; execute
+	; in a subroutine?
+	bit SOUND_SUBROUTINE, [hl] 
+	; execute
+	jr nz, .readcommand 
```

Comment:
If these comments weren't redundant, which they are, they should remain where they are. But the main problem is that the comments aren't needed in the first place — it should be obvious that `bit SOUND_SUBROUTINE, [hl]` is checking whether the engine is in a subroutine. These comments might've been necessary back when these were unnamed constants or simple numbers, but right now, that instruction is self-documenting, and so is the jump.